### DISCUSSION TITLE
Option to choose minimum client size

### DISCUSSION THREAD

**Discussion by TheArbalist** [runelite/runelite#15036] [Ideas]
We already have an option to lock the client, but could we get the option to specify a minimum size?

**Comment by alexanderhenne** [comment]
I guess you can set the client size in the RuneLite settings and then enable locking

**Comment by TheArbalist** [comment]
>I guess you can set the client size in the RuneLite settings and then enable locking

That's not setting the minimum size. That's setting it to *only* have that size.
Very different things.
